### PR TITLE
import subtomos from starfile

### PR DESCRIPTION
**Pull request by EstrellaFG**

### CODE REVIEW

File: .idea/scipion-em-pyseg.iml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (pconesa) [inline .idea/scipion-em-pyseg.iml:1]: I'd remove this file and gitignore .idea folder

File: pyseg/convert.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    item.setFileName(filename)
+    coordinate3d = Coordinate3D()
+    x = nline.split()[3]
+    y = nline.split()[4]
+    z = nline.split()[5]
+    coordinate3d.setX(float(x))
+    coordinate3d.setY(float(y))
+    coordinate3d.setZ(float(z))
+    ctf3d = nline.split()[1]
+    coordinate3d._3dcftMrcFile = String(join(path, ctf3d))
```
Comment (pconesa) [inline pyseg/convert.py:49]: Maybe comment here this name has to match the one that generates relion 3dctf?

File: setup.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+setup(
+    name='scipion-em-pyseg',  # Required
+    version='0.1',  # Required
+    description='Scipion plugin pyseg.',  # Required
+    long_description=long_description,  # Optional
+    url='https://github.com/scipion-em/scipion-em-pyseg',  # Optional
+    author='you',  # Optional
+    author_email='you@yourinstitution.email',  # Optional
+    keywords='scipion cryoem imageprocessing scipion-2.0',  # Optional
```
Comment (pconesa) [inline setup.py:34]: 3.0?

File: pyseg/convert.py (hunk 2 above)
Comment (EstrellaFG) [reply pyseg/convert.py:50]: This is already agreed with @JorMaister, but I've commented it anyway.

**Pull request merged**